cloudbuild.yaml

```diff
@@ -18,15 +18,11 @@ steps:
   - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20200713-e9b3d9d'
     entrypoint: make
     env:
-    - GIT_TAG=${_GIT_TAG}
     - PULL_BASE_REF=${_PULL_BASE_REF}
     - PROJECT=${_STAGING_PROJECT}
     args:
-    - build-image-and-push
+    - build-and-push-multi-arch
 substitutions:
-  # _GIT_TAG will be filled with a git-based tag for the image, of the form vYYYYMMDD-hash, and
-  # can be used as a substitution.
-  _GIT_TAG: '12345'
```

Comment:
does git_tag need to be plumbed through to STAGINGVERSION? I don't know if cloud build uses that when it releases (?)

Reply:
GIT_TAG seems to be no longer in use anymore, `STAGINGVERSION` reads from `PULL_BASE_REF`